dist/form-builder.js

```diff
@@ -1,2905 +1,2905 @@
+
+var fbUtils = {};
+
+// cleaner syntax for testing indexOf element
+fbUtils.inArray = function (needle, haystack) {
+  return haystack.indexOf(needle) !== -1;
+};
+
+/**
+ * Remove null or undefined values
+ * @param  {Object} attrs {attrName: attrValue}
+ * @return {Object}       Object trimmed of null or undefined values
+ */
+fbUtils.trimObj = function (attrs) {
+  var xmlRemove = [null, undefined, '', false, 'false'];
+  for (var attr in attrs) {
+    if (fbUtils.inArray(attrs[attr], xmlRemove)) {
+      delete attrs[attr];
+    } else if (Array.isArray(attrs[attr])) {
+      if (!attrs[attr].length) {
+        delete attrs[attr];
+      }
+    }
+  }
+
+  return attrs;
+};
+
+/**
+ * Test if attribute is a valid HTML attribute
+ * @param  {String} attr
+ * @return {Boolean}
+ */
+fbUtils.validAttr = function (attr) {
+  var invalid = ['values', 'enableOther', 'other', 'label',
+  // 'style',
+  'subtype'];
+  return !fbUtils.inArray(attr, invalid);
+};
+
+/**
+ * Convert an attrs object into a string
+ *
+ * @param  {Object} attrs object of attributes for markup
+ * @return {string}
+ */
+fbUtils.attrString = function (attrs) {
+  var attributes = [];
+
+  for (var attr in attrs) {
+    if (attrs.hasOwnProperty(attr) && fbUtils.validAttr(attr)) {
+      attr = fbUtils.safeAttr(attr, attrs[attr]);
+      attributes.push(attr.name + attr.value);
+    }
+  }
+  return attributes.join(' ');
+};
+
+/**
+ * Convert attributes to markup safe strings
+ * @param  {String} name  attribute name
+ * @param  {String} value attribute value
+ * @return {Object}       {attrName: attrValue}
+ */
+fbUtils.safeAttr = function (name, value) {
+  name = fbUtils.safeAttrName(name);
+  var valString = void 0;
+
+  if (value) {
+    if (Array.isArray(value)) {
+      valString = fbUtils.escapeAttr(value.join(' '));
+    } else {
+      if (typeof value === 'boolean') {
+        value = value.toString();
+      }
+      valString = fbUtils.escapeAttr(value.replace(',', ' ').trim());
+    }
+  }
+
+  value = value ? '="' + valString + '"' : '';
+  return {
+    name: name,
+    value: value
+  };
+};
+
+fbUtils.safeAttrName = function (name) {
+  var safeAttr = {
+    className: 'class'
+  };
+
+  return safeAttr[name] || fbUtils.hyphenCase(name);
+};
+
+/**
+ * Convert strings into lowercase-hyphen
+ *
+ * @param  {String} str
+ * @return {String}
+ */
+fbUtils.hyphenCase = function (str) {
+  str = str.replace(/[^\w\s\-]/gi, '');
+  str = str.replace(/([A-Z])/g, function ($1) {
+    return '-' + $1.toLowerCase();
+  });
+
+  return str.replace(/\s/g, '-').replace(/^-+/g, '');
+};
+
+/**
+ * convert a hyphenated string to camelCase
+ * @param  {String} str
+ * @return {String}
+ */
+fbUtils.camelCase = function (str) {
+  return str.replace(/-([a-z])/g, function (m, w) {
+    return w.toUp
```

Comment:
maybe we should just change this line to:
```javascript
if(!field.name){
  field.name = nameAttr(field);
}
```

ALso this change would need to be added to the src file [here](https://github.com/HayriCan/formBuilder/blob/master/src/js/form-builder.js#L535)